src/components/popover/popover.stories.ts

```diff
@@ -101,3 +101,29 @@ export const flipPlacements_TestOnly = (): string => html`
     document.querySelector(".my-popover").flipPlacements = ["right"];
   </script>
 `;
+
+export const scaleConsistencyPopoverHeadingActionSlottedIcon = stepStory(
```

Comment:
Per https://github.com/Esri/calcite-components/blob/master/conventions/Documentation.md#screenshot-test-only-stories, please add the `_TestOnly` suffix.